### PR TITLE
Stubbing out a file for Montreal

### PR DESCRIPTION
**Pull request by crazy4wisdom**
This is my extended message. PR resolves issue #51

### CODE REVIEW

File: crazy4wisdom-montreal.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Montreal, Canada
+
+### places to eat
+L'Avenue
+### Fun things to do
```
Comment (crazy4wisdom) [inline crazy4wisdom-montreal.md:7]: Add some fun things to do - can also add link to person by @

File: crazy4wisdom-montreal.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Montreal, Canada
+
+### places to eat
```
Comment (andymcf) [inline crazy4wisdom-montreal.md:3]: Hi from the UK

@crazy4wisdom

Comment (crazy4wisdom) [inline crazy4wisdom-montreal.md:3]: Hi @andymcf from Montreal, Canada

**Comment by crazy4wisdom** [comment]
This looks +1

**Pull request merged**